content/4. Deeltaken/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/Boolean/1. Boolean.md

```diff
@@ -0,0 +1,120 @@
+---
+title:
+  - 1. Boolean
+taxonomie:
+  - ib-19.2.Datatype-Primitief-Boolean.DT
+  - ib-19.3.Datatype-Primitief-Boolean.DT
+draft: false
+---
+
+> Meer informatie over [[4. Deeltaken/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/Boolean/1. Boolean|Boolean]]
+
+## Opdracht 1
+Maak een boolean aan en print het resultaat.
+
+### Specificaties
+- Creëer een `bool` genaamd `IsWaar`.
+- Print de waarde van de `IsWaar uit in de console.
```

Comment:
Opmerking: Deze specificaties lijken een andere insteek te hebben dan volgens het template hier: 
https://github.com/Windesheim-HBO-ICT/Leerlijn-Content-SE/blob/content/content/schrijfwijze/templates/DT%20template.md

Namelijk: in dit geval worden er stappen beschreven.  De template lijkt eisen voor te schrijven (dus bv 'er is een variabele van type boolean aangemaakt'). 

Ook niet onbelangrijk; er wordt gevraagd te bool te printen zonder deze eerst een waarde te geven. Wordt er al geacht te weten of een bool een default value heeft, en welke dit is? Of ontbreekt de stap om de waarde toe te kennen?

Reply:
Zelf heb ik de Specificaties heel erg vergeleken met de DT van If-else. Ik zal de verwoording wel wat meer aanpassen zodat het meer klinkt als een specificatie dan echt een beschreven stap!